### DISCUSSION TITLE
Auto-evaluate bind() method

### DISCUSSION THREAD

**Discussion by clementcelma** [alpinejs/alpine#4227] [3. Feature ideas]
In the same fashion as ``init()``, i wonder if it was possible to detect ``bind()`` method automatically.

**Comment by ekwoka** [comment]
It is possible.

It would more be about whether it makes enough sense to incorporate it into core as opposed to users just calling `this.bind()` in their init

**Reply by clementcelma** [reply]
I see. In that scenario, ``this.bind()`` should be called at the end of ``init()``, right ?

**Reply by ekwoka** [reply]
yes.
